dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -1110,6 +1111,8 @@ Make sure you create the following branch from the git tag (steps before) and no
 
 ```shell script
 cd ${AIRFLOW_REPO_ROOT}
+git checkout main
+git pull
```

Comment:
I asked @pierrejeambrun about this. he said we shouldn't do it from main branch but from the tag

Reply:
Hmm. That's strange - I'd say we should always run PRs from main :)

Reply:
Note the
`Make sure you create the following branch from the git tag (steps before) and not from main!` comment

Reply:
I don't think it is of big importance here because we fetch tags from remote repository.

But I agree that it order to get the latest source for the script running from main seems safer I would say.

Reply:
cool so if we want to run it from main we need to remove the comment as well

Reply:
Removed.